initfiles/componentfiles/configxml/espsmcservice.xsd.in

```diff
@@ -202,10 +202,10 @@
                 </xs:appinfo>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="logStashAdresses" type="xs:string" use="optional" default="">
+        <xs:attribute name="logStashAddresses" type="xs:string" use="optional" default="">
             <xs:annotation>
```

Comment:
@rpastrana @kunalaswani  This attribute is referenced in esp/services/ws_elk/WsELKService.cpp in the misspelled form